### DISCUSSION TITLE
Custom format for InvoiceID

### DISCUSSION THREAD

**Discussion by lukaskawerau** [btcpayserver/btcpayserver#2436] [Technical Support] [answered]
Is it possible to have InvoiceIDs have a custom format, such as having a certain prefix and be sequential?

In my jurisdiction, invoice ids are required by law to be sequential, and from screenshots it looks like BTCPayServer uses random strings as InvoiceIDs.

**Comment by Kukks** [comment, marked as answer]
> Is it possible to have InvoiceIDs have a custom format, such as having a certain prefix and be sequential?
> 
> In my jurisdiction, invoice ids are required by law to be sequential, and from screenshots it looks like BTCPayServer uses random strings as InvoiceIDs.

This is not possible. You can use the `orderId` property when creating an invoice though
